spectacles/cli.py

```diff
@@ -150,14 +153,32 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def set_file_handler(directory: str) -> None:
+
+    global LOG_FILEPATH
```

Comment:
Why does this need to be global? Only see it being used in this function.

Reply:
It's used in the exception handler to direct people to logs location.